### ISSUE TITLE
realm sdk mongodb client support

### ISSUE THREAD

**Issue by bshrram** [plexinc/papr#518]
I have application running in aws lambda serverless, the cold start takes a long time to setup connection pool, so using http requests to query data may be more suitable for serverless environments, and that can be done using [realm sdk mongodb client](https://www.mongodb.com/docs/realm/web/mongodb).

can papr fully support this client instead of the original mongodb connection?

**Comment by CarlosLozanoHealthCaters** [comment]
Have you followed the best practices to manage connection with AWS Lambda (https://www.mongodb.com/docs/atlas/manage-connections-aws-lambda/)?

The first point is probably the most important one!

**Comment by bshrram** [comment]
Yes sure, The problem is happening with cold start, these practices are good for warm requests.
and I want to add another point, managing number or connections of mongodb with serverless environment is hard and can reach the limit of connections easily, so moving that to data api is safer.

> Have you followed the best practices to manage connection with AWS Lambda (https://www.mongodb.com/docs/atlas/manage-connections-aws-lambda/)?
> 
> The first point is probably the most important one!
